modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/internal/type/test/EmailNotificationTypeTest.java

```diff
@@ -378,6 +396,32 @@ public void testFreeMarkerNotificationWithCommerceOrder() throws Exception {
 				TestPropsValues.getUserId()));
 	}
 
+	@Test
+	public void testFreeMarkerNotificationWithCustomService() throws Exception {
+		String body = LocalizationUtil.updateLocalization(
+			LocalizedMapUtil.getLocalizedMap(
+				HashMapBuilder.put(
+					LanguageUtil.getLanguageId(LocaleUtil.US),
+					"Test Service : ${objectFieldBusinessType}"
+				).build()),
+			null, "Body", LanguageUtil.getLanguageId(LocaleUtil.US));
```

Comment:
@rijema I don't understand why you need to use `LocalizationUtil.updateLocalization` here. Can you please explain?

Reply:
I was able to remove it. Please see my updates. Thanks.